### PR TITLE
Wrap and document PR #3314

### PR DESCRIPTION
**Pull request by lucatume**
This PR just wraps, adding documentation, [the PR from @mmcachran](https://github.com/moderntribe/the-events-calendar/pull/3314)
to add changelog entry and tracking for it.

### CODE REVIEW

File: src/Tribe/Aggregator/Cron.php [hunk 1]
```diff
@@ -192,18 +192,21 @@ public function action_register_cron() {
...
 	 * to make sure we can insert without having to expire cache.
 	 *
-	 * @param  array $schedules
+	 * @param array $schedules Current list of schedules.
 	 *
-	 * @return array
+	 * @return array            Modified list of schedules.
 	 */
-	public function filter_add_cron_schedules( array $schedules ) {
+	public function filter_add_cron_schedules( $schedules = [] ) : array {
```
Comment (Camwyn) [inline src/Tribe/Aggregator/Cron.php:199]:
```suggestion
	public function filter_add_cron_schedules( $schedules = [] ) {
```
That's a php 7+ syntax - unless we're now requiring php 7+ we shouldn't do this?

File: src/Tribe/Aggregator/Cron.php [hunk 2]
```diff
@@ -192,18 +192,21 @@ public function action_register_cron() {
...
+		// Ensure schedules is an array.
+		$schedules = is_array( $schedules ) ? $schedules : [];
```
Comment (Camwyn) [inline src/Tribe/Aggregator/Cron.php:201]: php 7

**Review by Camwyn — CHANGES_REQUESTED** [review]

File: src/Tribe/Aggregator/Cron.php (hunk 1 above)
Comment (lucatume) [reply src/Tribe/Aggregator/Cron.php:199]: Correct, thanks @Camwyn

**Review by Camwyn — APPROVED** [review]
Looks good 👍

**Pull request merged**